lib/ecto/adapters/mysql.ex

```diff
@@ -110,6 +110,32 @@ defmodule Ecto.Adapters.MySQL do
   @behaviour Ecto.Adapter.Storage
   @behaviour Ecto.Adapter.Structure
 
+  # Add dumpers functions if tagged values are supported by the driver
```

Comment:
We don't need the conditional here, instead we should change "mix.exs" to the minimum Mariaex version that supports this feature.